pkg/kubectl/cmd/apply.go

```diff
@@ -704,7 +704,7 @@ func (p *patcher) patchSimple(obj runtime.Object, modified []byte, source, names
 		return nil, nil, cmdutil.AddSourceToErr(fmt.Sprintf("getting instance of versioned object for %v:", p.mapping.GroupVersionKind), source, err)
 	case err == nil:
 		// Compute a three way strategic merge patch to send to server.
-		patchType = types.StrategicMergePatchType
+		patchType = types.MergePatchType
```

Comment:
this isn't correct... this is changing the patch type to merge patch, but continuing to compute a strategic merge patch below.